runelite-client/src/main/java/net/runelite/client/ui/components/PluginErrorPanel.java

```diff
@@ -49,11 +49,11 @@ public PluginErrorPanel()
 		setBorder(new EmptyBorder(50, 10, 0, 10));
 		setLayout(new BorderLayout());
 
-		noResultsTitle.setForeground(Color.WHITE);
+		noResultsTitle.setForeground(ColorScheme.TEXT_COLOR);
 		noResultsTitle.setHorizontalAlignment(SwingConstants.CENTER);
 
 		noResultsDescription.setFont(FontManager.getRunescapeSmallFont());
-		noResultsDescription.setForeground(Color.GRAY);
+		noResultsDescription.setForeground(ColorScheme.TEXT_COLOR.darker());
```

Comment:
GRAY. You basically consolidated LIGHT_GRAY_COLOR and GRAY what is not really the same I think